src/core/model.jl

```diff
@@ -9,6 +9,9 @@ SpaceType = Union{Nothing,AbstractSpace}
 
 abstract type DiscreteSpace <: AbstractSpace end
 
+abstract type AbstractPathfinder end
+PathfinderType = Union{Nothing,AbstractPathfinder}
```

Comment:
This has to go to the pathfinding file.

Reply:
This was kept here so in the future alternative pathfinding solutions like FMP in #412 can use it

Reply:
This is now moved to `spaces/pathfinding.jl`